src/HID-APIs/SurfaceDialAPI.h

```diff
@@ -32,12 +32,15 @@ typedef union{
 	uint8_t whole8[0];
 	uint16_t whole16[0];
 	uint32_t whole32[0];
+	uint64_t whole64[0];
```

Comment:
I believe these wholeX variables are actually not needed, see https://github.com/NicoHood/HID/issues/147
If you want you can remove them after testing.

Reply:
@hansmbakker Would you mind checking the project where those variables are used, and remove them entirely from the project? Maybe the uin8_t version could be kept with the name "raw", but a proper length value.

Reply:
@NicoHood what would the length be?